feg/gateway/services/testcore/ocs/mock_ocs/ccr_handler.go

```diff
@@ -323,3 +319,17 @@ func toGrantedUnitsAVP(resultCode uint32, validityTime uint32, quotaGrant *proto
 	}
 	return diam.NewAVP(avp.MultipleServicesCreditControl, avp.Mbit, 0, creditGroup)
 }
+
+func toGrantedServiceUnitAVP(quotaGrant *protos.Octets) []*diam.AVP {
```

Comment:
Can we pull this into a function tied to *protos.Octets? It looks like the implementation is the same for mock_ocs/mock_pcrf

Reply:
oh yes! i forgot that option

Reply:
I am not sure if it is the right way to add it to `go build mock_core_conversions.go`. In that file we have conversions between protos and specific types. But in this case we are converting proto into AVP. So we are extracting logic from the OCS and PCRF outside of the service. 

Also since they are test services, it may be beneficial to keep the code separate so we can run change them easily in the future.

Another solution we can do in the future is the creation of a shared library at the test_core folder with all the shared code we have on OCS and PCRF.... But again, being test services I think it is always better to have them separate so we are agile when we need to modify them.